RAMP/ramp.py

```diff
@@ -84,17 +84,18 @@ def unpack(bundle):
 @click.option('--redis-pack-min-version', '-R', 'redis_pack_min_version', default=module_metadata.MIN_REDIS_PACK_VERSION, help='redis pack minimum version')
 @click.option('--config-command', '-cc', default=module_metadata.CONFIG_COMMAND, help='command used to configure module args at runtime')
 @click.option('--os', '-O', default=module_metadata.OS, help='build target OS (Darwin/Linux)')
+@click.option('--os-list', callback=comma_seperated_to_list, help='build target OS (Darwin/Linux)')
```

Comment:
@rafie pls sync from master, we remove the os_list feature

Reply:
Also there are default values like author which are not None, would those always be override?